### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by lightingman117** [ep1cman/unifi-protect-backup#84] [General]
Just here to say thanks.

One of the drives in my UNVR died.
I bought a replacement and wanted to install it.
Found out it was in RAID5 and apparently that's not very good as it uses too much CPU for ARM.
Wanted to move to RAID10
Found out I'd lose all my footage :(

Found out there's a docker 'app' for Unraid...documentation was a bit hard to understand but I figured what the heck I'll try it anyways.

Remapped to a Disk Drive instead of appshare (NVMe)

I now have 60GB of backups for the past 30 days :)

It's not a full backup as it's only smart detection & AI
But 24TB of full backup would be a waste.

It was so easy and it just WORKS!!
Funny (Sad) how Ubiquiti can't do this.
I am HAPPY. Thank you!

**Comment by ep1cman** [comment]
Thanks for sharing, its nice to hear things work well and people appreciate them 😄
